src/Employees/Migrations/20221116090313_EmployeeEntityStartDate.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Employees.Migrations
+{
+    /// <inheritdoc />
+    public partial class EmployeeEntityStartDate : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTime>(
+                name: "StartDate",
+                table: "Employees",
+                type: "datetime2",
+                nullable: false,
+                defaultValue: new DateTime(1, 1, 1, 0, 0, 0, 0, DateTimeKind.Unspecified));
```

Comment:
Gir denne defaultValuen mening? 🤔

Reply:
Tenker det er greit med en default hvert fall. Kan endre året da, men greit at man ikke må ha samme mail på bemanning for å bli lagt til, så ser man at mailen ikke matcher hvis verdien er defualt